isdi

```diff
@@ -15,12 +15,11 @@ if __name__ == "__main__":
         print("Checking mode = {}\nApp flags: {}\nSQL_DB: {}"
               .format(config.TEST, config.APP_FLAGS_FILE,
                       config.SQL_DB_PATH))
-    print("TEST={}".format(config.TEST))
-    init_db(app, sa, force=config.TEST)
+    init_db(app, sa, force=False)
     handler = handlers.RotatingFileHandler('logs/app.log', maxBytes=100000, 
                                            backupCount=30)
     logger = logging.getLogger(__name__)
     logger.setLevel(logging.ERROR)
     logger.addHandler(handler)
-    PORT = 5000 if not config.TEST else 5002
+    PORT = 5000
```

Comment:
Is this change needed to solve #32? I think overall keeping configurability may be desirable.